doc/iterations/2024.3/shapes/improve_resizing_support_for_list_nodes.adoc

```diff
@@ -0,0 +1,38 @@
+= Improves resizing support for list nodes
+
+== Problem
+
+Until now, when a user resizes a list node, it is not functional.
+Only the container rendering is displayed during the resize action, the sub-container being stretched during the next layout.
+And it's not possible to reduce the size of the container (blocked by the size of the sub-elements).
+Sub-container can be resized in any direction, though it is up to the container to guarantee the width.
+
+== Key Result
+
+* When a container is resized, its sub-container must retain the rendering during the entire action.
+* Sub-container must be resizable vertically only.
+
+
+== Solution
+
+* When a resize event for a list node is triggered, create corresponding event to resize sub-elements.
+* Change sub-container node resizer to only allow vertical resize.
+
+== Scenario
+
+* Container resizing:
+
+image:images/list_node/container_resizing.gif[Container resizing]
+
+* Sub-container resizing:
+
+image:images/list_node/sub_container_resizing.gif[Sub-container resizing]
```

Comment:
Why isn't the blue getting smaller here? Why is the third one changing size?

Reply:
The specifier can now decide this behavior